tests/net/socket/offload_dispatcher/testcase.yaml

```diff
@@ -3,3 +3,5 @@ common:
 tests:
   net.socket.offload.dispatcher:
     tags: net socket
+    extra_configs:
+      - CONFIG_ZTEST_STACK_SIZE=8192
```

Comment:
I'd prefer to have this set in the `prj.conf`, it's not always the case that you run tests with twister.

Reply:
Hrm. testcase.yaml isn't referencing prj.conf at all; is there some implicit link between those two?

Reply:
`prj.conf` contains the default kconfig to be used when you build the app. You can override this with `CONF_FILE="..."`.

Reply:
I'm running tests on picolibc without this patch, given that there was already an increase in the stack size from d63cb793833e8bed30dda602bc0f045d84f6c91b

Reply:
appears to work fine with picolibc now, thanks for the help.

Reply:
pulled this patch as it's no longer required -- prj.conf sets the stack size to 2048 which is sufficient.